crates/pallet-subspace/src/lib.rs

```diff
@@ -18,6 +18,8 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![warn(unused_must_use, unsafe_code, unused_variables, unused_must_use)]
 
+extern crate alloc;
```

Comment:
Yes, I'm importing `String` from there below. I can't send `schnorrkel::SignatureError` through runtime interface due to the fact that it doesn't implement `Encode`/`Decode` and instead of mirroring `schnorrkel` enum with a separate enum just to be able to send it I decided to simply turn it into a string for now.